tests/clients/test_basic_client.py

```diff
@@ -61,6 +61,7 @@ def test_metrics_reporter_evaluate() -> None:
     test_current_server_round = 2
     test_loss = 123.123
     test_metrics: Dict[str, Union[bool, bytes, float, int, str]] = {"test_metric": 1234}
+    test_metrics_final: Dict[str, Union[bool, bytes, float, int, str]] = {"test_metric": 1234, "test - loss": 123.123}
```

Comment:
Two small recommendations:
(1) I think you can just redefine `test_metrics` here to be your new dictionary.
(2) I know you didn't write this, but I think that Union[bool, bytes, float, int, str] is equivalent to the Flower Scalar type. So you can just use that rather than the long union.